server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -55,6 +55,11 @@
   private static final int BATCH_SIZE = 80;
 
   /**
+   * The size of a batch of CVRs to be committed as a transaction.
+   */
+  private static final int TRANSACTION_SIZE = 400;
```

Comment:
Did you arrive at this number via testing?  Is there a relationship between average CVR size and some cache in the DB?  Something else?  (Magic numbers are scary.)

Reply:
I wanted it to be a multiple of the batch size, and something relatively small, and watching the timing displayed in logs made it seems like a reasonable choice. The reason it's a constant is so that we can tune it later if we need to.

Reply:
(in fact, I could see making it an actual properties setting at some point, along with batch size, so that it could be tuned based on the actual database in use)